reference/carts.v3.yml

```diff
@@ -1134,6 +1134,11 @@ components:
           description: 'The locale of the cart. Accepts strings of format `xx` or `xx-YY`. Uses the [ISO-639 standard](https://www.iso.org/iso-639-language-codes.html) format.'
           format: ISO-639
           example: en-US
+        source:
```

Comment:
Do we want to expose this publicly / allow developer to call V3 to create Manual Order directly?
I think this will be used by CP only so maybe we don't need to document it for now.

Also I would say it can be confusing to have this in the doc as we don't want people to believe they need to send the `source` on every requests, especially if we say we only accept `MANUAL_INCOMPLETE`. 

What do you think?

Reply:
yes that makes sense. I think again, that keep this a secret now should not cause any issue.